sqlitecluster/SQLite.cpp

```diff
@@ -430,7 +430,7 @@ bool SQLite::beginTransaction(SQLite::TRANSACTION_TYPE type) {
     // We actively track transaction counts incrementing and decrementing to log the number of active open transactions at any given moment.
     _sharedData.openTransactionCount++;
 
-    SINFO("Beginning transaction - open transaction count: " << (_sharedData.openTransactionCount));
+    SINFO("Beginning transaction - open transactions: " << (_sharedData.openTransactionCount));
```

Comment:
nab, i think it's actually helpful to say this is a count in the log line, otherwise it is hard to know what it is. If you see 0 or 1, could be bool; if you see some huge number, could be an ID. etc